sdk/eventhub/Microsoft.Azure.EventHubs/src/Amqp/AmqpExceptionHelper.cs

```diff
@@ -128,5 +128,18 @@ public static Exception ToMessagingContract(string condition, string message, bo
 
             return new EventHubsException(true, message);
         }
+
+        public static bool TryConvertToRetriableException(Exception exception, out EventHubsException retriableException)
```

Comment:
Should we consider adding tests for this?    Track Two uses a similar approach and has some [tests](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpExceptionExtensionsTests.cs#L144) for the scenario that may be easy to reflect to the T1 version.

Reply:
Good idea. I am adding some test coverage in the next iteration.